packages/jest-runner/corpus.ts

```diff
@@ -18,49 +18,57 @@ import path from "path";
 import fs from "fs";
 
 export class Corpus {
-	private readonly _inputDirectory: string;
-	private readonly _outputDirectory: string;
+	private readonly _inputsDirectory: string;
 
 	constructor(testPath: string, testStateElements: string[]) {
-		this._inputDirectory = this.buildInputDirectory(
+		this._inputsDirectory = this.buildInputsDirectory(
 			testPath,
 			testStateElements
 		);
-		this._outputDirectory = this.buildOutputDirectory(this._inputDirectory);
-		fs.mkdirSync(this._inputDirectory, { recursive: true });
-		fs.mkdirSync(this._outputDirectory, { recursive: true });
+		this.createMissingDirectoriesAndSeeds();
 	}
 
-	get inputDirectory(): string {
-		return this._inputDirectory;
+	get inputsDirectory(): string {
+		return this._inputsDirectory;
 	}
 
-	get outputDirectory(): string {
-		return this._outputDirectory;
+	inputsPaths(): [string, string][] {
+		return fs
+			.readdirSync(this._inputsDirectory)
+			.filter(
+				(entry) =>
+					!fs.lstatSync(path.join(this.inputsDirectory, entry)).isDirectory()
+			)
+			.map((file) => [file, path.join(this._inputsDirectory, file)]);
 	}
 
-	inputPaths(): [string, string][] {
-		return fs.readdirSync(this._inputDirectory).map((file) => {
-			return [file, path.join(this._inputDirectory, file)];
-		});
-	}
-
-	private buildOutputDirectory(inputDirectory: string): string {
-		return inputDirectory + path.sep;
-	}
-
-	private buildInputDirectory(
+	private buildInputsDirectory(
```

Comment:
Does it make sense to make `buildInputsDirectory` a private static method to make it clear that it is pure and does not depend on the object state?

Reply:
Good idea, pulled the function out of the `Corpus` class.